process/customer-success.md

```diff
@@ -0,0 +1,409 @@
+
+The content in this document is derived from past experiences as well as learning from the following online resources:
+
+* [https://www.helpscout.com/helpu/customer-support-department/](https://www.helpscout.com/helpu/customer-support-department/)
+* [https://www.zendesk.com/resources/structure-customer-support-organization/](https://www.zendesk.com/resources/structure-customer-support-organization/)
+* [https://www.mmcventures.com/wp-content/uploads/2018/05/MMC-Ventures-Customer-Service-Playbook-single-pages.pdf](https://www.mmcventures.com/wp-content/uploads/2018/05/MMC-Ventures-Customer-Service-Playbook-single-pages.pdf)
+
+# Customer Success with MayaData
+
+The PLOW culture at MayaData emphasizes that entire company MUST empathise with the Customer and make the Customer win within his/her Organization.  
+
+The single most important objective that the whole company strives for is to **_Deliver Data Agility - Free DevOps and Kubernetes from storage constraints using OpenEBS, validate with Litmus by chaos engineering and monitor with MayaOnline_***.*
+
+Our design philosophy is to deliver radically simple solutions that are open source based that enable developers and DevOps end user self service success.  We then sell solutions that wrap around these solutions to enable enterprises to control, manage, and scale their operations.  
+
+MayaData has chosen Chat as the primary mode of customer support, since Chat implicitly sets the expectations for a shorter response time compared to emails and also maintains a record of the interactions as opposed to Phone conversations. MayaData sets the bar for quick response very high compared to other companies - at less than 15 minutes.   Our goal is to be a trusted partner to our users - **_whether they are paid customer or community user_** - as they seek to achieve data agility.  Through this trusted relationship we can out learn alternatives while also building customer trust and loyalty.
+
+To achieve our rapid initial response SLA of less than 15 minutes, MayaData customer success organization is spread around the world - with at least one Tier 1 personnel on staff during work hours North America, Europe, and South Asia and Asia - so that users have access to customer success available 24 x 7.  A detailed schedule (*see Appendix : Support Shift Schedule* ) is published to maintain ownership of quick response. As the volume of the requests increase, the number of personnel available will be reassessed. 
+
+*Tier 1 customer success personnel are arguably the most important in MayaData when it comes to building a trusted relationship with end users and customers; these trusted relationships will, over time, result in customer growth and customer word of mouth which are the engines upon which the growth of MayaData is being built.  *
```

Comment:
Let us give a reference to the team structure somewhere over here so that the reader can jump to the section where the details are present.